farm/modeling/prediction_head.py

```diff
@@ -1044,7 +1044,7 @@ def logits_to_preds(self, logits, padding_mask, start_of_word, seq_2_start_t, ma
 
         # disqualify answers where end < start
         # (set the lower triangular matrix to low value, excluding diagonal)
-        indices = torch.tril_indices(max_seq_len, max_seq_len, offset=-1)
+        indices = torch.tril_indices(max_seq_len, max_seq_len, offset=-1, device="cuda")
```

Comment:
Would this work for a CPU? Maybe we could add the device as a property of PredictionHead.

Reply:
Yep, good point :+1: